services/minespace-web/src/components/pages/Project/ProjectDescriptionTab.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { shallow } from "enzyme";
+import { ProjectDescriptionTab } from "./ProjectDescriptionTab";
+import FeatureFlagContext from "@mds/common/providers/featureFlags/featureFlag.context";
+
+describe("ProjectDescriptionTab", () => {
+  it("renders properly", () => {
+    const component = shallow(
+      <FeatureFlagContext.Provider
+        value={{
+          isFeatureEnabled: () => true,
```

Comment:
We do already have in MS setupTests.js this function mocked. Not a big deal but it shouldn't be necessary.

Reply:
I think it would be worth switching out the shallow for the react-testing-library _render_. This one doesn't seem to be generating much of a snapshot.